src/binder/delete.rs

```diff
@@ -3,12 +3,13 @@ use super::*;
 impl Binder {
     pub(super) fn bind_delete(
         &mut self,
-        table_name: TableFactor,
+        tables: Vec<ObjectName>,
```

Comment:
This seems wrong. Guess the vec is dot separated name

Reply:
Nope. It is multiple tables. 😇
https://docs.rs/sqlparser/0.36.1/sqlparser/ast/enum.Statement.html#variant.Delete

Reply:
crazy 🤡